core-services/prow/02_config/_config.yaml

```diff
@@ -312,6 +312,15 @@ slack_reporter_configs:
     channel: '#ops-testplatform'
     report_template: Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}.
       <{{.Status.URL}}|View logs>
+  openshift-assisted/assisted-installer-deployment:
```

Comment:
Configuration looks good but we are trying to move away from a single huge config file like it used to be some times ago to a 'more distributed one', where every component holds its own. Try putting entries [here](https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml), even if I'm not sure we support `reporter_config` sharding we can give it a try for sure.

Reply:
yes, makes sense 
trying that now

Reply:
it's currently not handled by the ``shardprowconfig`` module